lib/YaST/NetworkSettings/HostnameDNSTab.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for Hostname/DNS Tab in YaST2
+# lan module dialog.
```

Comment:
To be consistent with other descriptions and how is invoked from cmd, 'yast2 lan' instead of 'YaST2 lan'.
(optional cosmetics): I've seen we align this text when have more than one line, just in case you want to follow that style.